stytch/b2b/recoverycodes.go

```diff
@@ -0,0 +1,107 @@
+package b2b
+
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/stytchauth/stytch-go/v11/stytch"
+	"github.com/stytchauth/stytch-go/v11/stytch/b2b/recoverycodes"
+	"github.com/stytchauth/stytch-go/v11/stytch/stytcherror"
+)
+
+type RecoveryCodesClient struct {
+	C stytch.Client
+}
+
+func NewRecoveryCodesClient(c stytch.Client) *RecoveryCodesClient {
+	return &RecoveryCodesClient{
+		C: c,
+	}
+}
+
+// Recover: Allows a Member to complete an MFA flow by consuming a recovery code. This consumes the
+// recovery code and returns a session token that can be used to authenticate the Member.
+func (c *RecoveryCodesClient) Recover(
+	ctx context.Context,
+	body *recoverycodes.RecoverParams,
+) (*recoverycodes.RecoverResponse, error) {
+	var jsonBody []byte
+	var err error
+	if body != nil {
+		jsonBody, err = json.Marshal(body)
+		if err != nil {
+			return nil, stytcherror.NewClientLibraryError("error marshaling request body")
+		}
+	}
+
+	headers := make(map[string][]string)
+
+	var retVal recoverycodes.RecoverResponse
+	err = c.C.NewRequest(
+		ctx,
+		"POST",
+		"/v1/b2b/recovery_codes/recover",
+		nil,
+		jsonBody,
+		&retVal,
+		headers,
+	)
+	return &retVal, err
+}
+
+// B2BGet: Returns a Member's full set of active recovery codes.
+func (c *RecoveryCodesClient) B2BGet(
+	ctx context.Context,
+	body *recoverycodes.B2BGetParams,
+) (*recoverycodes.B2BGetResponse, error) {
```

Comment:
Actually, a question: shouldn't this just be named `Get`? B2B seems redundant.

Reply:
Sure - I'm assuming this would just get overridden on the next autogen though. I'm not familiar if we have something in place to override naming